xds/src/main/java/io/grpc/xds/FaultFilter.java

```diff
@@ -205,7 +205,17 @@ public ClientInterceptor buildClientInterceptor(
       return null;
     }
     final Long finalDelayNanos = delayNanos;
-    final Status finalAbortStatus = abortStatus;
+    final Status finalAbortStatus;
+    if (abortStatus != null) {
```

Comment:
nit:  this can be more readable with a helper method:

```
final Status finalAbortStatus = getAbortStatusWithDescription(abortStatus);
...
private static Status  getAbortStatusWithDescription(Status abortStatus) {
  Status finalAbortStatus = null;
   if (abortStatus != null) {
      String abortDesc = "RPC terminated due to fault injection";
      if (abortStatus.getDescription() != null) {
        abortDesc = abortDesc + ": " + abortStatus.getDescription();
      }
      finalAbortStatus = abortStatus.withDescription(abortDesc);
    } 
   return finalAbortStatus;
}
```

Reply:
Done.